microapps/ArTabletopApp/app/src/main/java/com/arcgismaps/toolkit/artabletopapp/screens/MainScreen.kt

```diff
@@ -80,81 +100,110 @@ fun MainScreen() {
     val coroutineScope = rememberCoroutineScope()
 
     Box(modifier = Modifier.fillMaxSize()) {
-        TableTopSceneView(
-            arcGISScene = arcGISScene,
-            arcGISSceneAnchor = arcGISSceneAnchor,
-            translationFactor = 400.0,
-            modifier = Modifier.fillMaxSize(),
-            clippingDistance = 400.0,
-            tableTopSceneViewProxy = tableTopSceneViewProxy,
-            onInitializationStatusChanged = {
-                initializationStatus = it
-            },
-            onSingleTapConfirmed = { tap ->
-                arcGISSceneLayer.clearSelection()
-                coroutineScope.launch {
-                    identifiedBuilding = arcGISSceneLayer.identifyBuilding(
-                        tap.screenCoordinate,
-                        tableTopSceneViewProxy
-                    )
-                    identifiedBuilding?.let { identifiedBuilding ->
-                        arcGISSceneLayer.selectFeature(identifiedBuilding.feature)
-                    }
+        val sharedPreferences = LocalContext.current.getSharedPreferences("", Context.MODE_PRIVATE)
+        var acceptedPrivacyInfo by rememberSaveable { mutableStateOf(sharedPreferences.getBoolean(
+            KEY_PREF_ACCEPTED_PRIVACY_INFO, false)) }
+        var showPrivacyInfo by rememberSaveable { mutableStateOf(!acceptedPrivacyInfo) }
+        if (showPrivacyInfo) {
+            PrivacyInfoDialog(
+                acceptedPrivacyInfo,
+                onUserResponse = { accepted ->
+                    acceptedPrivacyInfo = accepted
+                    sharedPreferences.edit().putBoolean(KEY_PREF_ACCEPTED_PRIVACY_INFO, accepted).apply()
+                    showPrivacyInfo = false
+                }
+            )
+        }
+        if (!acceptedPrivacyInfo) {
+            Column(
+                modifier = Modifier.fillMaxSize(),
+                verticalArrangement = Arrangement.Center,
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                Text("Privacy Info not accepted")
+                Button(
+                    onClick = { showPrivacyInfo = true }
+                ) {
+                    Text("Show Privacy Info")
                 }
```

Comment:
Need to use string resources here:
```suggestion
                Text(stringResource(R.string.privacy_info_not_accepted))
                Button(onClick = { showPrivacyInfo = true }) {
                    Text(stringResource(R.string.show_privacy_info))
                }
```